static/sample-template-context.json

```diff
@@ -0,0 +1,224 @@
+{
+  "recipient" : {
+    "id" : "fd180cc2-a99c-4e43-b960-9b94daec5265",
+    "first-name" : "Joe",
+    "last-name" : "Dunphy",
+    "display-name" : "Joe D",
+    "public-data" : {
+      "phoneNumberVerified" : true
+    },
+    "protected-data" : {
+      "phoneNumber" : "123456"
+    },
+    "private-data" : {
+      "language" : "en"
+    }
+  },
+  "recipient-role" : "provider",
+  "other-party" : {
+    "id" : "ef7f40d5-da66-489a-957b-641313b68204",
+    "first-name" : "Jane",
+    "last-name" : "Pritchett",
+    "display-name" : "Jane P",
+    "public-data" : {
+      "phoneNumberVerified" : true
+    },
+    "protected-data" : {
+      "phoneNumber" : "789101112"
+    },
+    "private-data" : {
+      "language" : "fr"
+    }
+  },
+  "marketplace" : {
+    "name" : "Bikesoil",
+    "url" : "https://www.sharetribe.com"
+  },
+  "transaction" : {
+    "booking" : {
+      "start" : {
+        "year" : 2017,
+        "month" : 12,
+        "day" : 1,
+        "hours" : 0,
+        "minutes" : 0,
+        "seconds" : 0,
+        "milliseconds" : 0
+      },
+      "end" : {
+        "year" : 2017,
+        "month" : 12,
+        "day" : 12,
+        "hours" : 0,
+        "minutes" : 0,
+        "seconds" : 0,
+        "milliseconds" : 0
+      },
+      "displayStart" : {
+        "year" : 2017,
+        "month" : 12,
+        "day" : 1,
+        "hours" : 0,
+        "minutes" : 0,
+        "seconds" : 0,
+        "milliseconds" : 0
+      },
+      "displayEnd" : {
+        "year" : 2017,
+        "month" : 12,
+        "day" : 12,
+        "hours" : 0,
+        "minutes" : 0,
+        "seconds" : 0,
+        "milliseconds" : 0
+      }
+    },
+    "listing" : {
+      "id" : "d892dfe6-9407-4400-9e81-8100d7a87b7f",
+      "title" : "Wooden sauna",
+      "private-data" : { },
+      "public-data" : { },
+      "metadata" : { },
+      "availability-plan" : {
+        "type" : "availability-plan/time",
+        "timezone" : "Europe/Helsinki"
+      }
+    },
+    "delayed-transition" : {
+      "run-at" : {
+        "year" : 2017,
+        "month" : 11,
+        "day" : 29,
+        "hours" : 0,
+        "minutes" : 0,
+        "seconds" : 0,
+        "milliseconds" : 0
+      }
+    },
+    "payout-total" : {
+      "amount" : 810,
+      "currency" : "USD"
+    },
+    "protected-data" : { },
+    "customer" : {
+      "id" : "ef7f40d5-da66-489a-957b-641313b68204",
+      "first-name" : "Jane",
+      "last-name" : "Pritchett",
+      "display-name" : "Jane P",
+      "public-data" : {
+        "phoneNumberVerified" : true
+      },
+      "protected-data" : {
+        "phoneNumber" : "789101112"
+      },
+      "private-data" : {
+        "language" : "fr"
+      }
+    },
+    "payin-total" : {
+      "amount" : 900,
+      "currency" : "USD"
+    },
+    "id" : "d8893061-c860-4e44-8d4d-61587a497e6e",
+    "line-items" : [ {
```

Comment:
Should this be `tx-line-items`?

Reply:
Yes, line-items is deprecated. The format of tx-line-items also differs a bit. Do we use the line-items version when testing and rendering at server side?

Reply:
Yes, we use `line-items` in the Build API when validating the templates. This context was the same as is used there.

Reply:
Looking more closely at the whole sample context JSON now, and I noticed that it has both `line-items` as well as `tx-line-items`, so this was probably left there for backwards compatibility with old templates. Should be fine as-is IMO.